app/api.py

```diff
@@ -799,16 +799,26 @@ def get_element_text(self, doc, tag_name):
         else:
             return None
 
-    # ga_send_hit sends hit to Google Analytics via Measurment Protocol.
-    # With Measurement Protocol, you can send data by making HTTP requests.
-    # You can find more on developer guide.
-    # https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide
-    def ga_send_hit(self, event_category):
-        ga_cid = uuid.uuid4()
+    def send_hit_to_google_analytics(self, event_category):
+        """Sends hit to Google Analytics via Measurment Protocol.
+        With Measurement Protocol, you can send data by making HTTP requests.
+        You can find more on developer guide.
+        https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide
+        Args:
+            event_category: Google Analytics Event Category.
+        """
+
+        # Check Google Analytics property setting
+        try:
```

Comment:
I believe ```self.env.analytics_id``` just returns None instead of throwing exception. So you should check if the returned value is None instead of using try-except.

Also, when ```self.env.analytics_id``` is none, just don't send Measurement Protocol request, instead of using a specific default ID. It would be consistent behavior with normal Google Analytics usage of Person Finder:
https://github.com/google/personfinder/blob/1390a96cfabc0d5458354760c202de74610abb13/app/resources/analytics.html.template#L13
And we don't want to log traffic to our Analytics ID for non-production Person Finder.